packages/web-components/src/components/content-block-card-static/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+
+# `<dds-content-block-card-static>`
```

Comment:
@IgnacioBecerra can you also add documentation here about how to enable using the feature flag `DDS_CONTENT_BLOCK_CARD_STATIC`?